static/docs/commands-reference/repro.md

```diff
@@ -1,7 +1,7 @@
 # repro
 
-Reproduce a stage that is specified by a `.dvc` file and all stages the file
-lists as dependencies (recursively).
+Rerun all commands recorded in the pipeline in the same order, 
+or rerun specific commands associated with stages named on the command line.
```

Comment:
One fundamental things are missing which makes this description not informative:  repro checks and reproduces recursively all changed stages. Semantics is similar to how `make` works. Can we take a look at the description of make and get some ideas?

Also, I don't think the `reproduce` word is bad. Why did we decide to change it? Let's assume I have a `model.pkl` and a `model.dvc` files. Basically, `model.dvc` describes how to reproduce the `model.pkl` and dvc takes care to run necessary steps.

Reply:
To define the "repro" command as "reproducing" is a circular definition